packages/regl-worldview/src/commands/GLText.js

```diff
@@ -416,9 +449,21 @@ function makeTextCommand() {
 }
 
 export default function GLText(props: Props) {
+  const context = useContext(WorldviewReactContext);
+  const { dimension } = context;
+
   const [command] = useState(() => makeTextCommand());
   // HACK: Worldview doesn't provide an easy way to pass a command-level prop into the regl commands,
   // so just attach it to the command object for now.
   command.autoBackgroundColor = props.autoBackgroundColor;
+
+  command.scaleInvariant = !!props.scaleInvariantFontSize;
+  // Compute the actual size for the text object in NDC coordinates (from -1 to 1)
+  // In order to make sure the text is always shown at the same size regardless of
+  // the canvas dimensions (as Text does), we need to scale it based on both the desired
+  // font size and the current worldview resolution. Otherwise, the text will be
+  // displayed at different sizes depending on the worlview canvas dimension.
+  command.scaleInvariantSize = (props.scaleInvariantFontSize ?? 0) / dimension.height;
```

Comment:
Can we add a test for changing the dimensions of the canvas after it's been rendered once to make sure that this still works?

Reply:
Done :)